docs/spec/v1beta2/ocirepositories.md

```diff
@@ -593,6 +602,53 @@ instance hosted at [rekor.sigstore.dev](https://rekor.sigstore.dev/).
 Note that keyless verification is an **experimental feature**, using
 custom root CAs or self-hosted Rekor instances are not currently supported.
 
+#### Notation
+
+The `notation` provider can be used to verify the signature of an OCI artifact using known
+trust policy and CA certificate.
+
+```yaml
+---
+apiVersion: source.toolkit.fluxcd.io/v1beta2
+kind: OCIRepository
+metadata:
+  name: <repository-name>
+spec:
+  verify:
+    provider: notation
+    secretRef:
+      name: notation-public-keys
+```
+
+When the verification succeeds, the controller adds a Condition with the
+following attributes to the OCIRepository's `.status.conditions`:
+
+- `type: SourceVerified`
+- `status: "True"`
+- `reason: Succeeded`
+
+To verify the authenticity of an OCI artifact, create a Kubernetes secret
+with the Notation CA certificate and trust policy:
+
+```yaml
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: notation-public-keys
+type: Opaque
+data:
+  certificate1.pem: <BASE64>
```

Comment:
Do you think we should a reference to [Signing and Verification Workflow](https://github.com/notaryproject/specifications/blob/v1.0.0/specs/signing-and-verification-workflow.md)? It might help users in deciding which option to choose

Reply:
Great idea! Resolved in [f0db7d1](https://github.com/fluxcd/source-controller/pull/1075/commits/f0db7d1e68652ff8673e3e349f0d3a10cea89e93) and [14b7710](https://github.com/fluxcd/source-controller/pull/1075/commits/14b7710d0e66ea41ea925076461fbe3dd89e212c)